addons/point_of_sale/i18n/point_of_sale.pot

```diff
@@ -5885,3 +5885,10 @@ msgstr ""
 #, python-format
 msgid "were duplicates of existing orders"
 msgstr ""
+
+#. module: point_of_sale
+#. openerp-web
+#: code:addons/point_of_sale/static/src/js/Screens/TicketScreen/TicketScreen.js:0
+#, python-format
+msgid "%s has a total amount of %s, are you sure you want to delete this order ?"
```

Comment:
Thanks but there is also the "Existing orderlines" parts that's missing. Also we try to keep the entries in alphabetical order (to avoid two people adding the same term at different places). The easiest is to go to [your runbot instance](https://runbot.odoo.com/runbot/r-d-1?search=81065) and export the .pot file from there (filtering out the irrelevant parts if needed)

Reply:
Done, thanks